### ISSUE TITLE
Predefine sort order for certain keys.

### ISSUE THREAD

**Issue by MasterEric** [Gudahtt/prettier-plugin-sort-json#31]
Create a new option which includes an array of strings. If a given key is in the array, it will be displayed before the others, in the order specified in the configuration. Otherwise, the key will be displayed after.

Example:
```yaml
jsonSortForceOrder:
  - f
  - o
  - u
  - b
  - a
  - r
```
This JS object:
```
{
  a: null,
  b: null,
  f:  null,
  z: null,
  k: null,
  u: null,
  o: null,
  r: null, 
}
```
Would be sorted like so:
```
{
  f: null,
  o: null,
  u:  null,
  b: null,
  a: null,
  r: null,
  k: null,
  z: null, 
}
```

This can already be done with #30 (as seen with the `foubar` text case) but this would provide a more convenient means of performing this same function without having to create an additional file.

**Comment by Gudahtt** [comment]
Interesting idea! This sounds useful.

One possible enhancement would be to use a special syntax to denote classes of characters, like regex character classes maybe. So then you could pass in this:

```
jsonSortForceOrder:
  - f
  - \d
  - r
```

to force the object `{ r: null, f: null, 9: null }` to be sorted as `{ f: null, 9: null, r: null }`.

Other nice-to-have qualities would be:
- The ability to match _one character_ in a class or any number of characters in a class
- The ability to specify that a class of characters should be sorted in _reverse_ order
- Maybe more advanced regex syntax?
- What happens when classes overlap? This should be made clear

**Comment by Gudahtt** [comment]
This is now supported by the `jsonSortOrder` option implemented in #92. Let me know what you think!
